src/gui/optionsdlg.cpp

```diff
@@ -95,7 +99,8 @@ OptionsDialog::OptionsDialog(QWidget *parent)
 #endif
     m_ui->tabSelection->item(TAB_ADVANCED)->setIcon(GuiIconProvider::instance()->getIcon("preferences-other"));
     for (int i = 0; i < m_ui->tabSelection->count(); ++i) {
-        m_ui->tabSelection->item(i)->setSizeHint(QSize(std::numeric_limits<int>::max(), 64));  // uniform size for all icons
+        // uniform size for all icons
+        m_ui->tabSelection->item(i)->setSizeHint(QSize(std::numeric_limits<int>::max(), 62));
```

Comment:
To let it fit the window without showing a vertical scrollbar. See https://github.com/qbittorrent/qBittorrent/pull/6952/files/8f48837ed9fe4799c9f917ff22eab6226371d03d#diff-2c91a19e5c2af0562178cfdaacafa3ec.
I fixed the above one as well, was probably due to an automatic merge of some PR.